Install issues due to Python version

Error details:
```% npm install                                                                                                                                                                                       (10010)

> keytar@4.2.1 install /Users/zhauser/code/twilio-cli/node_modules/keytar
> prebuild-install || node-gyp rebuild

prebuild-install WARN install No prebuilt binaries found (target=10.9.0 runtime=node arch=x64 platform=darwin)
gyp ERR! configure error
gyp ERR! stack Error: Python executable "/usr/local/opt/python/libexec/bin/python" is v3.6.5, which is not supported by gyp.
gyp ERR! stack You can pass the --python switch to point to Python >= v2.5.0 & < 3.0.0.
gyp ERR! stack     at PythonFinder.failPythonVersion (/Users/zhauser/.nvm/versions/node/v10.9.0/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:501:19)
gyp ERR! stack     at PythonFinder.<anonymous> (/Users/zhauser/.nvm/versions/node/v10.9.0/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:483:14)
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:280:7)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at maybeClose (internal/child_process.js:961:16)
gyp ERR! stack     at Socket.stream.socket.on (internal/child_process.js:380:11)
gyp ERR! stack     at Socket.emit (events.js:182:13)
gyp ERR! stack     at Pipe._handle.close (net.js:599:12)
gyp ERR! System Darwin 17.7.0
gyp ERR! command "/Users/zhauser/.nvm/versions/node/v10.9.0/bin/node" "/Users/zhauser/.nvm/versions/node/v10.9.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/zhauser/code/twilio-cli/node_modules/keytar
gyp ERR! node -v v10.9.0
gyp ERR! node-gyp -v v3.7.0
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! keytar@4.2.1 install: `prebuild-install || node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the keytar@4.2.1 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/zhauser/.npm/_logs/2018-09-28T16_15_37_944Z-debug.log
```

My system is a Mac. With my `PATH`, `python` refers to python3. The only way I can get python2 at this point is directly running `/usr/bin/python`. Unfortunately I don't recall how I got my system to this state. It might have just been `brew install python` (which is now python3) though.

Possible fixes:
- `npm install --python=/path/to/python2` (tested and works)
- have a `python2` command, e.g. if (in my case) I'd run `ln -s /usr/bin/python /usr/local/bin/python2` (not tested, but I saw some docs about `gyp` now looking for `python2` first)

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     "@oclif/plugin-autocomplete": "^0.1.0",
     "@oclif/plugin-help": "^2.1.6",
     "@oclif/plugin-plugins": "^1.7.7",
-    "@twilio/cli-core": "^0.0.1",
+    "@twilio/cli-core": "^1.0.0",
     "chalk": "^2.4.1",
     "ngrok": "^3.0.1",
     "twilio": "^3.30.2"
@@ -21,7 +21,7 @@
   "devDependencies": {
     "@oclif/dev-cli": "^1.21.0",
     "@oclif/test": "^1.1.0",
-    "@twilio/cli-test": "^0.0.1",
+    "@twilio/cli-test": "^1.0.0",
     "chai": "^4.1.2",
     "eslint": "^4.19.1",
     "eslint-config-oclif": "^1.5.1",

--- a/src/commands/project/add.js
+++ b/src/commands/project/add.js
@@ -49,6 +49,7 @@ class ProjectAdd extends BaseCommand {
     this.authToken = this.flags['auth-token'];
     this.projectId = this.flags.project;
     this.force = this.flags.force;
+    this.region = this.flags.region;
   }
 
   validateAccountSid() {
@@ -108,7 +109,8 @@ class ProjectAdd extends BaseCommand {
 
   getTwilioClient() {
     return twilio(this.accountSid, this.authToken, {
-      httpClient: new CLIRequestClient(this.id, this.logger)
+      httpClient: new CLIRequestClient(this.id, this.logger),
+      region: this.region
     });
   }
 
@@ -140,7 +142,7 @@ class ProjectAdd extends BaseCommand {
       return;
     }
 
-    this.userConfig.addProject(this.projectId, this.accountSid);
+    this.userConfig.addProject(this.projectId, this.accountSid, this.region);
     await this.secureStorage.saveCredentials(this.projectId, apiKey.sid, apiKey.secret);
     const configSavedMessage = await this.configFile.save(this.userConfig);
 
@@ -164,6 +166,9 @@ ProjectAdd.flags = Object.assign(
     force: flags.boolean({
       char: 'f',
       description: 'Force overwriting existing project credentials'
+    }),
+    region: flags.string({
+      hidden: true
     })
   },
   TwilioClientCommand.flags // Yes! We _do_ want the same flags as TwilioClientCommand

--- a/src/commands/project/list.js
+++ b/src/commands/project/list.js
@@ -5,9 +5,14 @@ class ProjectList extends BaseCommand {
   async run() {
     await super.run();
     if (this.userConfig.projects.length > 0) {
+      // If none of the projects have a region, delete it from all of them so it doesn't show up in the output.
+      if (!this.userConfig.projects.some(p => p.region)) {
+        this.userConfig.projects.forEach(p => delete p.region);
+      }
+
       this.output(this.userConfig.projects);
     } else {
-      this.logger.warn('No projects have been configured. Run ' + chalk.whiteBright('twilio login') + ' to add one!');
+      this.logger.warn('No projects have been configured. Run ' + chalk.whiteBright('twilio project:add') + ' to add one!');
     }
   }
 }

--- a/test/commands/project/add.test.js
+++ b/test/commands/project/add.test.js
@@ -8,9 +8,9 @@ const helpMessages = require('../../../src/services/messaging/help-messages');
 describe('commands', () => {
   describe('project', () => {
     describe('add', () => {
-      const addTest = test
+      const addTest = (commandArgs = []) => test
         .twilioCliEnv(Config)
-        .twilioCreateCommand(ProjectAdd, [])
+        .twilioCreateCommand(ProjectAdd, commandArgs)
         .do(ctx => {
           const fakePrompt = sinon.stub();
           fakePrompt
@@ -27,7 +27,7 @@ describe('commands', () => {
           ctx.testCmd.exit = sinon.fake();
         });
 
-      addTest
+      addTest()
         .nock('https://api.twilio.com', api => {
           api.get(`/2010-04-01/Accounts/${constants.FAKE_ACCOUNT_SID}.json`).reply(200, {
             sid: constants.FAKE_ACCOUNT_SID
@@ -53,7 +53,7 @@ describe('commands', () => {
           );
         });
 
-      addTest
+      addTest()
         .nock('https://api.twilio.com', api => {
           api.get(`/2010-04-01/Accounts/${constants.FAKE_ACCOUNT_SID}.json`).reply(500, {
             error: 'oops'
@@ -67,7 +67,7 @@ describe('commands', () => {
           expect(ctx.stderr).to.contain('Could not validate the provided credentials');
         });
 
-      addTest
+      addTest()
         .nock('https://api.twilio.com', api => {
           api.get(`/2010-04-01/Accounts/${constants.FAKE_ACCOUNT_SID}.json`).reply(200, {
             sid: constants.FAKE_ACCOUNT_SID
@@ -82,6 +82,24 @@ describe('commands', () => {
           await ctx.testCmd.run();
           expect(ctx.stdout).to.equal('');
           expect(ctx.stderr).to.contain('Could not create an API Key');
+        });
+
+      addTest(['--region', 'dev'])
+        .nock('https://api.dev.twilio.com', api => {
+          api.get(`/2010-04-01/Accounts/${constants.FAKE_ACCOUNT_SID}.json`).reply(200, {
+            sid: constants.FAKE_ACCOUNT_SID
+          });
+          api.post(`/2010-04-01/Accounts/${constants.FAKE_ACCOUNT_SID}/Keys.json`).reply(200, {
+            sid: constants.FAKE_API_KEY,
+            secret: constants.FAKE_API_SECRET
+          });
+        })
+        .stdout()
+        .stderr()
+        .it('supports other regions', async ctx => {
+          await ctx.testCmd.run();
+          expect(ctx.stdout).to.equal('');
+          expect(ctx.stderr).to.contain('configuration saved');
         });
     });
   });

--- a/test/commands/project/list.test.js
+++ b/test/commands/project/list.test.js
@@ -27,6 +27,24 @@ describe('commands', () => {
         .it('runs project:list with 1 project', ctx => {
           expect(ctx.stdout).to.contain('default');
           expect(ctx.stdout).to.contain(constants.FAKE_ACCOUNT_SID);
+          expect(ctx.stdout).to.not.contain('Region');
+          expect(ctx.stderr).to.equal('');
+        });
+
+      test
+        .do(ctx => {
+          ctx.userConfig = new ConfigData();
+          ctx.userConfig.addProject('default', constants.FAKE_ACCOUNT_SID, 'dev');
+        })
+        .twilioCliEnv(Config)
+        .stdout()
+        .stderr()
+        .twilioCommand(ProjectList, [])
+        .it('runs project:list with 1 regional project', ctx => {
+          expect(ctx.stdout).to.contain('default');
+          expect(ctx.stdout).to.contain(constants.FAKE_ACCOUNT_SID);
+          expect(ctx.stdout).to.contain('dev');
+          expect(ctx.stdout).to.contain('Region');
           expect(ctx.stderr).to.equal('');
         });
     });

